crates/autopilot/src/domain/eth/mod.rs

```diff
@@ -1,5 +1,21 @@
 pub use primitive_types::{H160, U256};
 
+/// An address. Can be an EOA or a smart contract address.
+#[derive(Debug, Default, Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Hash)]
+pub struct Address(pub H160);
+
+impl From<H160> for Address {
```

Comment:
Both `From<>` can be derived with `derive_more`

Reply:
fixed